.pipelines/arc-register-extension-all-regions-stable.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+# Register azuremonitor-containers extension with Arc Registration API
+
+REGISTER_REGIONS_CANARY=$REGISTER_REGIONS_CANARY
+RELEASE_TRAINS_PREVIEW_PATH=${RELEASE_TRAINS_PREVIEW_PATH}
+RELEASE_TRAINS_STABLE_PATH=${RELEASE_TRAINS_STABLE_PATH}
+IS_CUSTOMER_HIDDEN=$IS_CUSTOMER_HIDDEN
+
+PACKAGE_CONFIG_NAME="${PACKAGE_CONFIG_NAME:-microsoft.azuremonitor.containers-pkg022022}"
+API_VERSION="${API_VERSION:-2021-05-01}"
+METHOD="${METHOD:-put}"
+REGISTRY_PATH_CANARY_PREVIEW="https://mcr.microsoft.com/azuremonitor/containerinsights/canary/preview/azuremonitor-containers"
+REGISTRY_PATH_CANARY_STABLE="https://mcr.microsoft.com/azuremonitor/containerinsights/canary/stable/azuremonitor-containers"
+REGISTRY_PATH_PROD_STABLE="https://mcr.microsoft.com/azuremonitor/containerinsights/prod1/stable/azuremonitor-containers"
+
+REGISTER_REGIONS_BATCH=($REGISTER_REGIONS_BATCH)
+
+echo "Start arc extension registration, REGISTER_REGIONS is $REGISTER_REGIONS_CANARY, RELEASE_TRAINS are $RELEASE_TRAINS_PREVIEW_PATH; $RELEASE_TRAINS_STABLE_PATH, PACKAGE_CONFIG_NAME is $PACKAGE_CONFIG_NAME, API_VERSION is $API_VERSION, METHOD is $METHOD"
```

Comment:
this is not registration, rather this is release, right?

Reply:
changed